doc/Admin Documentation.md

```diff
@@ -191,4 +191,15 @@ Interactive admin configuration:
 occ twofactorauth:gateway:configure sms
 ```
 
+### Plivo
+URL: https://www.plivo.com
+Stability: Experimental
+
+Use the HTTPS service provided by clicksend.com for sending SMS.
```

Comment:
clicksend still? ;)

Reply:
Copy-and-paste error...will fix in next commit.